lib/web_ui/lib/src/engine/keyboard.dart

```diff
@@ -5,6 +5,7 @@
 // @dart = 2.6
 part of engine;
 
+const Duration _keydownCancelDuration = Duration(seconds: 1);
```

Comment:
After doing some research on "the internet" it looks like the typical ranges for linux and windows go up to 1000ms. On my Mac it can go up to 2000ms.

I'll update our threshold to 2500ms to accommodate all the typical ranges. I'll add a doc comment as well.